db/content.sql

```diff
@@ -0,0 +1,9 @@
+/*
+    This file contants inital content for any tables.
+
+    The table must be first defined in schema.sql before any content
+    is added!
+*/
+insert into consent_type (id, shortname, description, enabled, protect, privacypref)
+    values (1, 'ENROLL', 'General terms-of-use for this BOINC project.', 0, 1, 0),
+    (2, 'STATSEXPORT', 'Do you consent to exporting your data to BOINC statistics aggregation Web sites?', 0, 1, 1);
```

Comment:
Since consent_type.id is auto-increment and you are intentionally not tying specific values of consent_type.id to behavior in the application, then you should not specify the value for the id on the insert.  

```
insert into consent_type (shortname, description, enabled, protect, privacypref)
    values ('ENROLL', 'General terms-of-use for this BOINC project.', 0, 1, 0),
    ('STATSEXPORT', 'Do you consent to exporting your data to BOINC statistics aggregation Web sites?', 0, 1, 1);
```

Reply:
Fixed in 4cb30b1